internal/testrunner/runners/system/servicedeployer/terraform.go

```diff
@@ -42,6 +44,36 @@ type TerraformServiceDeployer struct {
 	definitionsDir string
 }
 
+// addTerraformOutputs method reads the terraform outputs generated in the json format and
+// adds them to the custom properties of ServiceContext and can be used in the handlebars template
+// like `{{TF_OUTPUT_queue_url}}` where `queue_url` is the output configured
+func addTerraformOutputs(customProps map[string]interface{}) error {
```

Comment:
Can you add a unit test for the function. If this accepted a file path as an argument then you could more easily construct a unit test for it.

Reply:
Fixed..